### PR TITLE
Created migration and seeder

### PR DESCRIPTION
**Pull request by ovpdng124**
ready

### CODE REVIEW

**Comment by ovpdng124** [comment]
ready

File: app/Model/Task.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Model;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = ['name'];
```
Comment (quoctd-0607) [inline app/Model/Task.php]: protected $fillable = [
'name',
];

**Comment by ovpdng124** [comment]
ready

**Comment by ovpdng124** [comment]
ready

**Comment by ovpdng124** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**